feature-tests/tests/resources/step-definitions/invoke-api-gateway-happy-path.step.ts

```diff
@@ -52,6 +52,86 @@ defineFeature(feature, (test) => {
     );
   });
 
+  test('Happy Path - Suspended Account - Get Request to /ais/userId - Returns Expected Data for <aisEventType>', ({
+    given,
+    when,
+    then,
+  }) => {
+    given(
+      /^I send an (.*) intervention message to the TxMA ingress SQS queue for a Account in (.*) state$/,
+      async (originalAaisEventType, aisEventType) => {
+        await sendSQSEvent(testUserId, originalAaisEventType);
+        await sendSQSEvent(testUserId, aisEventType);
```

Comment:
Could it be maybe prudent to put a wait between these two steps?

Reply:
Good idea Chiara, ill make that change and update the PR, thank you

Reply:
![image](https://github.com/govuk-one-login/account-interventions-service/assets/110972905/1a28f938-d36e-45b4-9456-bc787af6fd32)
changes made and all tests still passing, thank you